en/datenshi_beatmap_criteria_en.md

```diff
@@ -2,5 +2,122 @@
 title: "Beatmap Rules (English)"
 old_id: 20
 ---
+#FAQ
+A few question that are frequently asked
 
-Edit disini bos
+-----------------------
+
+- **Q: My map hasn't been ranked but it follows the criteria?**
+- A: There's a chance that your map doesn't get detected by the system, you can re-request again in the discord channel '#beatmap-requests` **Only for Donators**
+
+-----------------------
+
+- **Q: Why isn't my map submitted?**
+- A: That can happen when : The beatmap is new, wrong beatmap link, or the beatmap database retrieval system from Datenshi to official osu! has not been updated 
+- 
+-----------------------
+
+- **Q: How long does it take for my map to be ranked?**
+- A: Depending on the Beatmap Nominator themselves, BN's need time to review and sort out beatmaps that follow the criteria 
+
+-----------------------
+
+- **Q: How many maps can I request at this time?**
+- A: For now the maximum request is 4 (This might change from time to time) 
+
+-----------------------
+
+
+#### Notes
+Before requesting, make sure your unranked map you are submitting are :
+
+- In Graveyard, at least 1 month to avoid the map getting updated by the mapper (abandoned map) 
+- Metadata that soothe the eyes. For example **Diao Ye Zong feat.** is bad, please avoid it. 
+
+<h1><i class="comment icon"></i> Standard</h1>
+
+#### Rules
+
+1. The map must be playable and still follows the Bancho's RC. If not, it is likely to be loved. 
+2. A longstream map that is easily abused is likely to be loved . (Because the majority of relax players can use the DT mod to get PP)
```

Comment:
```suggestion
2. Longstream maps that are easy to abuse are likely to be loved. (Because many relax players can use the DT mod to get PP)
```

Reply:
```suggestion
2. Longstream maps that are easy to abuse are likely to be loved. (Because the majority of relax players can use the DT mod to get PP)
```
pakai majority aja karena kalau pakai many kurang jelas.